spark-gremlin/src/test/java/org/apache/tinkerpop/gremlin/spark/structure/SparkTest.java

```diff
@@ -51,6 +51,15 @@
  * @author Marko A. Rodriguez (http://markorodriguez.com)
  */
 public class SparkTest extends AbstractSparkTest {
+    @Test
+    public void testCustomizedSparkAppName() {
```

Comment:
Minor nit: could add a similar test to make sure that the default name is "Apache TinkerPop's Spark-Gremlin"

Reply:
Good idea. I'll add one.